tests/mocks.py

```diff
@@ -0,0 +1,109 @@
+class MockConsoleIO:
```

Comment:
I like that you've moved the definitions for all your mocks into one file. It makes it easy to find them and to support reuse throughout the test-codebase. :+1:

Do you think there are any downsides to this approach?